frontend/webpack-stats.json

```diff
@@ -1 +1 @@
-{"status":"done","publicPath":"http://127.0.0.1:8080/","chunks":{"app":[{"name":"bundle.js","publicPath":"http://127.0.0.1:8080/bundle.js","path":"C:\\Users\\Tristan\\CS467 Capstone\\frontend\\dist\\bundle.js"}]}}
\ No newline at end of file
+{"status":"done","publicPath":"http://127.0.0.1:8080/","chunks":{"app":[{"name":"bundle.css","publicPath":"http://127.0.0.1:8080/bundle.css","path":"D:\\467 capstone\\Capstone\\capstone\\frontend\\dist\\bundle.css"},{"name":"bundle.js","publicPath":"http://127.0.0.1:8080/bundle.js","path":"D:\\467 capstone\\Capstone\\capstone\\frontend\\dist\\bundle.js"},{"name":"app.51764a541f6f2427a4d8.hot-update.js","publicPath":"http://127.0.0.1:8080/app.51764a541f6f2427a4d8.hot-update.js","path":"D:\\467 capstone\\Capstone\\capstone\\frontend\\dist\\app.51764a541f6f2427a4d8.hot-update.js"}]},"error":"ModuleError","message":"Module Error (from ./node_modules/eslint-loader/index.js):\nerror: Parsing error: Unexpected token, expected \")\"\n\n  102 |        console.log(this)\n  103 |       // Tell the REST API to create or update a Post Instance\n> 104 |       if ((this.post_body && this.post_body.length > 0) && (speech && speech.length > 0) {\n      |                                                                                          ^\n  105 |         this.error = \"Please use text-to-speech or type your message, not both.\";\n  106 |       } else if (!this.post_body && speech.length < 1) {\n  107 |         this.error = \"You can't send an empty post!\"; at src\\components\\Application\\Posts\\views\\PostEditor.vue:162:89:\n  160 |        console.log(this)\n  161 |       // Tell the REST API to create or update a Post Instance\n> 162 |       if ((this.post_body && this.post_body.length > 0) && (speech && speech.length > 0) {\n      |                                                                                         ^\n  163 |         this.error = \"Please use text-to-speech or type your message, not both.\";\n  164 |       } else if (!this.post_body && speech.length < 1) {\n  165 |         this.error = \"You can't send an empty post!\";\n\n\n1 error found."}
```

Comment:
Might not want to merge this into master either.

Reply:
I'll remove it from the branch